README.md

```diff
@@ -41,11 +41,36 @@ as well as performing operations like minting, swapping, and burning
 1. **Using Factory and Pair Contracts Only**  
    This demo handles deploying the Factory and Pair contracts and executing a complete flow of operations  
    [View the demo task](https://github.com/NilFoundation/uniswap-v2-nil/blob/main/tasks/core/demo.ts)
+   
+   **Important:**
+   - The `UniswapV2Pair` is deployed on the same shard as the `UniswapV2Factory`.
```

Comment:
Not sure i understand this? Our factory works that it can support any shard

Reply:
agree, removed